client/stylesheets/_modal.scss

```diff
@@ -921,7 +925,87 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Report Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Button ~~~~*/
+
+.modReportTrigger {
+	background-color: #fff !important;
+	color: #404040 !important;
+}
+
+.modReportTrigger:hover {
+	background-color: #d3d3d3 !important;
+}
 
+/*~~ Button End ~~*/
+
+/*~~~~ Listing ~~~~*/
+
+.modReportWarning {
+	margin-top: 60px !important;
+	margin-right: auto !important;
+	margin-left: auto !important;
+	width: 80%;
+}
+
+.modReportWarning img {
+	border-radius: 50%;
+}
+
+.modReportWarning li:first-child, .modReportWarning li:last-child {
+	vertical-align: middle !important;
+}
+
+.modReportWarning li:last-child {
+	margin-left: 5px !important;
+}
+
+.modReportPageTwo ul li {
+	font-size: 1.2rem;
+	margin: 10px 0;
+	padding: 5px 0;
+}
+
+.modReportPageTwo ul li:last-child {
```

Comment:
Merge rule `.modReportPageTwo ul li:last-child` with rule on line 851<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `modReportPageTwo` should be written in lowercase with hyphens